frontend/packages/kubevirt-plugin/src/utils/common.ts

```diff
@@ -2,7 +2,16 @@ import * as _ from 'lodash';
 
 export const omitEmpty = (obj, justUndefined = false) => {
```

Comment:
isn't there a `lodash` alternative so we can remove this code?

Reply:
have not found one

Reply:
[`_.compact(array)`](https://lodash.com/docs/4.17.15#compact)

> Creates an array with all falsey values removed. The values `false`, `null`, `0`, `""`, `undefined`, and `NaN` are falsey.

Reply:
Alternatively, you could use `_.remove` or `_.without`.

Reply:
Cannot be used very well. It should work on objects and arrays recursively with an option to switch between null x undefined